data/dapis/polygon-testnet.json

```diff
@@ -1,32 +1,25 @@
 {
   "AAVE/BTC": "0x09a5873667837598bd0990ba2f53d750d545ce435ecdcd44e0b4c64ab7d7d20d",
-  "AAVE/USD": "0xf418b8f3ee3f8b66946bce48d56d69f031a977c0b6a700b07452eb1da9ad5bab",
-  "ADA/USD": "0x8dd297503e6e8b95c979677e9e6aecfee08ee5426bea72adff2d7a797f7bd69d",
-  "ALPACA/USD": "0xdc82ce0ab839eb9b3f65309c0c81019dc90c2260f34e6de4d157d5975049bfd0",
-  "AVAX/USD": "0x098cf42f8ea393f32ec9b788d48eb58d044a51a8793f1d98ceb27d25fccbeb2b",
-  "AXS/USD": "0x2b6c4fdf87eb7e4321115c5e04dc6dd2ae3a4e83f443c1c9f493eaa98b192a59",
-  "BNB/USD": "0x9a8f1b8d4cb1dc23d1d8a5991770b6fc220b524c4a3f11a9ea5579a3e0cef367",
-  "BTC/USD": "0x0a6d78c904e255189a08bfd0773f6fc9da2c5a02a65aa16c8ec83dd346825fe4",
-  "BUSD/USD": "0x7f6a0ef132e1283d5905030ddf555ff2d96c0ab5f7de64653241d5fd93a7b96e",
-  "CAKE/USD": "0x6bd5cb95e7e3784f5467b48c7711ee7adbc859a94cef67fcc23084ce50120b82",
-  "CRV/USD": "0x985cedbc20e6f3a3eeebcd2af8a0266a8b5da285e3ef87964aa4d96f4c4729ab",
-  "DOT/USD": "0xc8d57dc4972714cc8754fda28fdd824faf63607b26fa7bc895006400c7298093",
-  "EOS/USD": "0xddd22962d8a815f315667f4bd8dfceb04a4383cd3a95397806345840d2705197",
-  "FIL/USD": "0x71f4095b47160ae099768e2920d0377ca0fb01d6adf6a5ee269c90020202c511",
-  "JOE/USD": "0x2811b7988f3c239d7f7f988523e795f7cc0f34e45ae0bca304e4782d49448833",
-  "LINK/USD": "0x29e1bb7ebdc04ae43bf4a598aaaaf6f2de29ef49ed71865a38166f5d6bbea76e",
-  "LTC/USD": "0x79e3268fa44de913c648260d7d483875ab0095ea18d63be088ef6894bd8bcacd",
-  "MATIC/USD": "0x5f9d68d2108556123524a6d64531bb136676e2ca6effda812cca70e5ef655528",
-  "MIOTA/USD": "0xf94ee8a799f14c1c4900335d4eb73395e224a6cf8ec3fde7734227235a5bd1c8",
-  "NEO/USD": "0xb43559f0ce38988d695fa1a07becebc6add4295ff086fd326d831be92067c240",
-  "OOKI/ETH": "0xc22d30e582730222b2f3227f357452c2275520339e76d831784ed484493b3b1b",
```

Comment:
Will OOKI/ETH also be decommissioned? [comment on #114](https://github.com/api3dao/operations/pull/114#issuecomment-1212025556)

Reply:
Yes, I missed it then